tfhe/src/shortint/server_key/scalar_bitwise_op.rs

```diff
@@ -57,7 +57,6 @@ impl ServerKey {
     }
 
     // by convention smart operations take mut refs to their inputs, even if they do not modify them
-    #[allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
why are these ones gone but the comment remains ?

Reply:
I missed it
Will remove them